libs/component-sample/src/lib/feature/search-result-list-sample/search-result-card-sample.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, ChangeDetectorRef } from '@angular/core';
+
+@Component({
+  selector: 'gsa-sam-search-result-card-sample',
+  templateUrl: './search-result-card-sample.component.html'
+})
+export class SearchResultCardSampleComponent  {
+
+  constructor(private change: ChangeDetectorRef) { }
+
+  items = [
+    { iconlib : 'sds', icon: "bell", text: 'First', id: 1 },
+    { iconlib : 'sds', icon: 'book', text: 'Second', id: 2 },
+    { iconlib : 'sds', icon: 'download', text: 'Third', id: 3 },
+    { iconlib : 'fas', icon: 'comment', text: 'Fourth', id: 4 },
+    { iconlib : 'sds', icon: 'arrow-left', text: 'Fifth', id: 5 , hasNewerData:true},
```

Comment:
The result item model is probably not going to return from the API or server with the icon library and icon needed. It is more likely that it will return an item **type** that we then need to interpret. As an example it would be better to have a small method in the component class that converts a type to the icon values needed.

Example data: { type: 'article', text: 'Entity Help', id:1} 

Create a method to take article type, find the corresponding mapping and then return { iconLib: 'sds', icon: 'book' } - And then use the returned value in the template.